tox.ini

```diff
@@ -56,6 +56,7 @@ exclude = .venv,.tox,dist,doc,*egg,.git,build,tools
 basepython = python3
 deps =
     doc8
+    m2r
```

Comment:
I don't think ReadTheDocs uses `tox` to generate the docs; do we need to put `m2r` somewhere else too, so that RTD will install it?

Reply:
Ah good catch. It appears that sphinx can use a requirements.txt file or a conf.py [1]. Today we point at the requirements.txt file so cloud-init can be installed, which seems unnecessary.

I think a doc-requirements.txt file makes sense and having tox point at that like we do for integration and tests. I'll then update readthedocs look for that file.

[1] https://docs.readthedocs.io/en/stable/guides/specifying-dependencies.html#

Reply:
added the doc-requirements.txt in latest push